### ISSUE TITLE
after "O.K. you can start talking!" the The phone was hung up immediately

### ISSUE THREAD

**Issue by mikeyang01** [twilio-samples/speech-assistant-openai-realtime-api-node#18]
any ideas to dig or debug this issue? No error log appears

**Comment by bnb** [comment]
#15 solves this, but also we don't have error logging so issues might be missed if you change the code. If you'd like error logging to be included by default, PRs welcome!

**Comment by holographix** [comment]
Sorry to revamp this issue, but I found myself stuck within the same problem, and I wanted to add my story in the hope it might help others unstuck themselves. 

I had the repo cloned on my local machine, followed along with the tutorial and everything worked like butter. 
Then I tried to simply put the code on my server to ditch ngrok and being able to call from a regular phone without the need of using the dev-phone. 
That moment this issue started to preset itself. 
Took me a bit to figure out that the problem was that since I was running the code on my server, I tried to hook that up to twilio using the raw [ip address:port] format all via http ( NO HTTPS ), the same went for the websocket, I changed the prefix from `wss` to `ws` and this was the root of the issue. 
Once I put an https certificate and, a proper dns and reverted everything back to https / wss it started working like a charm.  

**TLDR; Service won't work on a server without https certificate ( like certbot / let's encrypt )**
